File: Printpress.Application/SpareParts/DTO/SparePartItemDto.cs
namespace Printpress.Application;

public record SparePartItemDto
{
    public Guid Id { get; init; }
    public string Name { get; init; }
    public int? PacksPerCarton { get; init; }
    public int? UnitsPerPack { get; init; }
    public int StockQuantity { get; init; }
    public int TotalInQuantity { get; init; }
    public int TotalOutQuantity { get; init; }
    public bool HasTransactions { get; init; }
}

File: Printpress.UI/Printpress_Angular/src/app/features/spare-parts/models/spare-part-item.dto.ts
export interface SparePartItemDto {
  id: string;
  name: string;
  packsPerCarton: number | null;
  unitsPerPack: number | null;
  stockQuantity: number;
  totalInQuantity: number;
  totalOutQuantity: number;
  hasTransactions: boolean;
}
